src/openqaoa-core/openqaoa/algorithms/workflow_properties.py

```diff
@@ -329,6 +343,45 @@ def __init__(
 
         self.calibration_data_location = calibration_data_location
 
+class Mitiq_Zne_ErrorMitigationProperties(ErrorMitigationProperties):
+    """
+    choose ZNE from Mitiq as your error mitigation technique for the QAOA circuit.
+
+    Parameters
+    ----------
+    error_mitigation_technique: str
+        The name of the error mitigation technique.
```

Comment:
Add the `str` that triggers creation of a `Mitiq_Zne_ErrorMitigationProperties` object.

Reply:
I don't understand what modifications to perform. Please verify if the last commits addressed this issue as well

Reply:
I meant describing which input `error_mitigation_technique` takes here, but that may be redundant.